### ISSUE TITLE
Invalid prop `accessibilityRole` of value `tab` supplied to `TouchableOpacity`

### ISSUE THREAD

**Issue by emmarq** [react-native-community/react-native-tab-view#955]
### Current behaviour

The use of `TabBarItem` is throwing: 
```
Failed prop type: Invalid prop `accessibilityRole` of value `tab` supplied to `TouchableOpacity`, expected one of 
["none","button","link","search","image","keyboardkey","text","adjustable","imagebutton","header","summary"]
```

### Expected behaviour

The correct display of the MaterialTopTabNavigator.

### Code sample

https://github.com/react-native-community/react-native-tab-view/blob/c762699b01c34adcd80b4761e120ebe0573ba649/src/TabBarItem.tsx#L220

### Screenshots (if applicable)

<img width="559" alt="Screen Shot 2020-01-23 at 2 24 36 PM" src="https://user-images.githubusercontent.com/2946454/73016998-fa5a0080-3dec-11ea-8596-41d7815375ae.png">
<img width="515" alt="Screen Shot 2020-01-23 at 2 24 52 PM" src="https://user-images.githubusercontent.com/2946454/73016999-fa5a0080-3dec-11ea-9467-31e4cb9fcc76.png">

### What have you tried

I can't find a way to overwrite that prop. 

### Your Environment

| software                     | version
| ---------------------------- | -------
| android               | 24
| ios | 13.3
| react-native                 | expo 35, react-native 0.59
| react-navigation-tabs        | 2.6.2
| react-native-gesture-handler | 1.3.0
| react-native-reanimated      | 1~.2.0
| node                         | 12.13.1
| npm or yarn                  | 6.12.1

**Comment by emmarq** [comment]
I hack this around installing the dependency directly at 2.11.0.

**Comment by AbdelkhalekESI** [comment]
same issue

**Comment by satya164** [comment]
Please upgrade your React Native version.

**Comment by selvamariappant2s** [comment]
> I hack this around installing the dependency directly at 2.11.0.

save my day :)

**Comment by ankushdec1993** [comment]
Also facing the same issue in RN 0.59.0.

**Comment by ankushdec1993** [comment]
Is it possible to make it work without downgrading react-native-tab-view.

**Comment by grit96** [comment]
@satya164 it would be useful to have React Native version compatibility information in the README.

**Comment by satya164** [comment]
PR welcome to add it

**Comment by MMelek00** [comment]
> I hack this around installing the dependency directly at 2.11.0.

sorry what you mean by that?

**Comment by emmarq** [comment]
> > I hack this around installing the dependency directly at 2.11.0.
> 
> sorry what you mean by that?

Installing the dependency directly at 2.11.0 fixed my issue. 

What I think was happening was that one of my dependencies has as a dependency on this library, but with a bad model version that was installing an incompatible code with my current setup. So that update should have been a major, or the dependency should have specified the correct model version. (By model version, I mean the ~, or ^, etc). I hope I made myself clear. 

It is not the first time something crash in my projects because of one transitive update.

**Comment by grit96** [comment]
@satya164 is there any way backwards compatibility for this prop can be added for RN < 0.60?

**Comment by nicewahson** [comment]
same issue. And 2.11.0 doesn't show anything~

**Comment by Iamdhammi** [comment]
> I hack this around installing the dependency directly at 2.11.0.

Thanks this worked for me :)

**Comment by karthikvn23** [comment]
which component should be 2.11.0 ?

**Comment by roryabraham** [comment]
Having this same issue...Also not sure which dependency should be 2.11.0

**Comment by codeforcesmeme** [comment]
@satya164  I think you can look into it? provide backward compatibility ?
You're just breaking many projects

**Comment by Magudeshwaran** [comment]
> I hack this around installing the dependency directly at 2.11.0.
can you explain that ?

**Comment by Magudeshwaran** [comment]
> > > I hack this around installing the dependency directly at 2.11.0.
> > 
> > 
> > sorry what you mean by that?
> 
> Installing the dependency directly at 2.11.0 fixed my issue.
> 
> What I think was happening was that one of my dependencies has as a dependency on this library, but with a bad model version that was installing an incompatible code with my current setup. So that update should have been a major, or the dependency should have specified the correct model version. (By model version, I mean the ~, or ^, etc). I hope I made myself clear.
> 
> It is not the first time something crash in my projects because of one transitive update.

Can you specify exactly what you updated ?

**Comment by Magudeshwaran** [comment]
Go to node_modules -> react-natie-tab-view -> lib -> module 
Change 'accessibilityRole' value to 'none' in 'TabBarItem.js' and 'TabBar.js'  
Solved for me.

**Comment by YobertyGarcia** [comment]
Have the same issue, the only way it got solved was manually changing the lines described by Magudeshwaran on May 23, 

the lines 156 on TabBarItem.js and 241 on TabBar.js 

This is not the better solution but it works
